tests/framework/resources.py

```diff
@@ -94,15 +94,19 @@ def get_stats(self):
     @staticmethod
```

Comment:
the commit says that these changes will go away once we release 0.24.1. Is that so? Wouldn't the integration tests use snapshots into both 0.24.0 and 0.24.1?

Reply:
They will go away for sure. Once we release the next v0.24 we will replace the binaries used in this test (we will use the latest v0.24 binary available) and this is the same version (luckily) which introduced the balloon device. Snapshots are unaffected, you can create and resume a snapshot in any v0.24 variant.

Reply:
I see now, I confused the binary with the snapshot artifacts

Reply:
let's open a issue to track removing these pls

Reply:
Issue [linked here](https://github.com/firecracker-microvm/firecracker/issues/2530).